python/cudf/tests/test_dataframe.py

```diff
@@ -1062,3 +1062,17 @@ def test_binops(pdf, gdf, left, right, binop):
     d = binop(left(pdf), right(pdf))
     g = binop(left(gdf), right(gdf))
     assert_eq(d, g)
+
+
+@pytest.mark.parametrize('func', [
+    lambda df: df.empty,
+    lambda df: df.x.empty,
+])
+def test_unary_operators(func, pdf, gdf):
+    p = func(pdf)
+    g = func(gdf)
+    assert_eq(p, g)
+
+
+def test_is_monotonic(gdf):
+    assert hasattr(gdf.index, 'is_monotonic')
```

Comment:
Is this test a placeholder for future monotonic tests? Otherwise I don't see much value for checking if the attribute exists as a test.

Reply:
I'd be happy dropping the test if desired.

Reply:
I've changed this to a more sensible test